models/articles_models.js

```diff
@@ -35,4 +35,14 @@ function checkArticleIDExists(article_id){
     })
 }
 
-module.exports = { fetchArticleById, fetchAllArticles,checkArticleIDExists }
\ No newline at end of file
+function updateArticleById(article_id, inc_votes){
+    return Promise.all([db.query(`UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *`, [inc_votes,article_id]),checkArticleIDExists(article_id)])
```

Comment:
for clarity it would be nicer to have your query separated so its nicer to read here!